src/main/webapp/transcript.js

```diff
@@ -1,9 +1,84 @@
+
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const TRANSCRIPT_CONTAINER = 'transcript-container';
+const ENDPOINT_TRANSCRIPT = '/transcript';
+
 /**
  * Sends a POST request to the transcript.
  */
 function sendPostToTranscript() {
-  // TODO: Update with user input.
-  const testParamsString = 'id=123456789&video=8PrOp9t0PyQ';
-  const params = new URLSearchParams(testParamsString);
-  fetch('/transcript', {method: 'POST', body: params});
+  // TODO: Update with user input and URL Builder.
+  const paramsString = 'id=123456789&video=3ymwOvzhwHs';
+  const params = new URLSearchParams(paramsString);
+  fetch(ENDPOINT_TRANSCRIPT, {method: 'POST', body: params})
+      .then(fetchTranscriptLines(paramsString));
+}
+
+/**
+ * Fetches the transcript lines from {@code ENDPOINT_TRANSCRIPT}.
+ *
+ * <p>{@code paramsString} indicates the video ID
```

Comment:
Why not rename paramsString to videoId. Seems more descriptive. paramsString does not indicate much about what it is.

Reply:
I forgot to mention that the string also includes the lecture ID. I updated the param description to reflect this.